airbyte-integrations/connectors/source-mailchimp/sample_files/state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Campaigns": {"create_time": "2020-11-23T05:42:11+00:00"},
```

Comment:
is it really `create_time` and not `created_time`?

Reply:
yup!